kalite/distributed/features/superuser_create.feature

```diff
@@ -2,58 +2,47 @@ Feature: Create superuser from the in browser modal
 
     Scenario: Superuser already exists
         Given I am on the homepage
+        And there is superuser 
```

Comment:
This step no longer exists, I removed it since a superuser always exists. (It's created in base_environment, from your last PR.)